### ISSUE TITLE
Label file

### ISSUE THREAD

**Issue by JavierGranado** [TensorBox/TensorBox#234]
JavierGranado @JavierGranado 00:01
Hi, I have retrained Tensorbox to detect “objects” diferent of people and it works fine. Now I have the pre-trained model file (.pb) but I do not know how to get the label file (.txt).
I would like to deploy my tensorbox retrained model on the Tensorflow Android Camera Demo. I have yet modified the app to work only with the TF_Detect “localize and track people in the camera preview in real-time” and it works on my smatrphone, but it crash when I replace the .pb original file for my .pb and I don’t know how can I replace the “multibox_location_priors.txt” which is the original file. What is more, I’m not sure what is the meaning of this file, because is not a label file in the sense of classification model which includes all of the outputs labels of the net.
Can you give some advice about it.
Thanks in advance.
